shell.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Import ENV variables
+set -o allexport
+source .env
+set +o allexport
+
+ID=`${PLATFORM} container ls -f name=${STACKNAME}_api_sails --format={{.ID}}`
```

Comment:
Is this only allowing us to go into `api_sails`?  How about if I want to get to the cli of another service?

Reply:
This script was added to `ab-production-runtime` at some point. I didn't modify this (and don't plan to). For me the underlying commands are easy enough that I don't think scripting it is worthwhile. But, if someone else wants to improve this they can.